tests/acceptance/test_node_operators_registry.py

```diff
@@ -60,6 +68,10 @@ def test_finalize_upgrade(contract):
             CURATED_STAKING_MODULE_STUCK_PENALTY_DELAY,
             {"from": contracts.voting},
         )
+    with reverts("UNEXPECTED_CONTRACT_VERSION"):
```

Comment:
Could you please clarify what specifically needs to be cleaned up?

This test is expected to function correctly after the vote, and it will throw an error if anyone attempts to call `finalizeUpgrade_v3` on `NodeOperatorsRegistry` contract, the same logic as  in`finalizeUpgrade_v2` check in that test a few lines above.

Ideally, the test should be removed only when the `finalizeUpgrade_v2` and `finalizeUpgrade_v3` methods are removed from the `NodeOperatorsRegistry`  contract in a future upgrade.